schemas/usage.json

```diff
@@ -45,7 +45,7 @@
 			"type": "object",
 			"patternProperties": {
 				".*\\.php$": {
-					"$id": "#/system/plugins/active_plugins/$plugin",
+					"$id": "#/definitions/plugins/plugin",
```

Comment:
Why?

Reply:
It was added with wrong *id* in the past, i noticed it here and fixed.